src/vconsole/vconsole-setup.c

```diff
@@ -195,9 +195,15 @@ static void font_copy_to_all_vcs(int fd) {
         unsigned char map8[E_TABSZ];
         unsigned short map16[E_TABSZ];
         struct unimapdesc unimapd;
-        struct unipair unipairs[USHRT_MAX];
+        _cleanup_free_ struct unipair* unipairs = NULL;
         int i, r;
 
+        unipairs = malloc(sizeof(struct unipair) * USHRT_MAX);
+        if (unipairs == NULL) {
+                log_error("Not enough memory to copy fonts");
```

Comment:
That's a reasonable change, considering that we're allocating couple hundred kb here.

Please change this to use `new`.